docs/08-plugins/vault.md

```diff
@@ -0,0 +1,147 @@
+# vault  Plugin
+
+This plugin installs [vault](https://www.vaultproject.io/) in an existing Kubernetes cluster using the Helm chart for you test or develop vault.
+This plugin installs vault with replicas:3 by default value
+
+### Usage
+
+```yaml
+tools:
+- name: hashicorp-vault
+  # name of the plugin
+  instanceID: default
+  options:
+    # need to create the namespace or not, default: false
+    create_namespace: true
+    repo:
+      # name of the Helm repo
+      name: hashicorp
+      # url of the Helm repo
+      url: https://helm.releases.hashicorp.com
+    # Helm chart information
+    chart:
+      # name of the chart
+      chart_name: hashicorp/vault
+      # release name of the chart
+      release_name: vault
+      # k8s namespace where Vault will be installed
```

Comment:
Here the "Vault" is "correct," consistent with the official name. Please update all the other occurrences of "Vault" ("vault" isn't the official name.)

Similarly, if "GitHub" is used, make sure it's not spelled as "github."

Reply:
If use "Vault" for release_name, helm API will throw: Secret "sh.helm.release.v1.Vault.v1" is invalid: metadata.name: Invalid value: "sh.helm.release.v1.Vault.v1": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')

Here, use lowercase instead.